atlas/packages/app/src/shared/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { css } from '@emotion/core'
+import * as Icons from '../../icons'
+import { camelCase } from 'lodash'
+
+type IconType = Icons.IconType
+type IconProps = {
+  icon: IconType
```

Comment:
Maybe `name` would be more fitting since we already have `Icon` as the component name?